source/slang/slang-ir-insts.h

```diff
@@ -410,9 +408,13 @@ struct IRLookupWitnessMethod : IRInst
 {
     IRUse witnessTable;
     IRUse requirementKey;
+    IRUse interfaceType;
 
     IRInst* getWitnessTable() { return witnessTable.get(); }
     IRInst* getRequirementKey() { return requirementKey.get(); }
+    IRInst* getInterfaceType() { return interfaceType.get(); }
+
+    IR_LEAF_ISA(lookup_interface_method)
```

Comment:
It really shouldn't be required for a `lookup_interface_method` to have the interface type as an explicit operand.

A better solution in the long run would be to have an `IRWitnessTableType` that takes the interface being conformed to as an operand (e.g., `WitnessTable<IFoo>`). In that case, the `lookup_interface_method` instruction could have an invariant that the `witnessTable` operand needs to have an `IRWitnessTableType`, and in that case it could produce the interface type by looking at the operand of the witness table's type.

Reply:
Removed, I just need to handle the `TransitiveSubtypeWitness` lowering case and ensure a correct `IRWitnessTable` type is emitted there so I don't get nullptr for the type of the resulting `lookup_interface_method` inst.